src/styles/components/_search-bar.scss

```
@@ -41,16 +45,29 @@
 
 .c-search-bar--lg.c-tags-input {
 	.c-tags-input__control {
-		min-height: 6.4rem;
-		border-radius: 3.2rem;
+		min-height: 4.8rem;
+		border-radius: 2.4rem;
+
+		@include respond-at(md) {
+			min-height: 6.4rem;
+			border-radius: 3.2rem;
+		}
 	}
 
 	.c-tags-input__value-container {
-		padding-left: $spacer-lg;
+		padding-left: $spacer-xs;
+
+		@include respond-at(md) {
+			padding-left: $spacer-lg;
+		}
 	}
 
 	.c-tags-input__indicators {
-		padding-right: $spacer-md;
+		padding-right: 1.4rem;
```

Persoonlijk normaliseer ik waarden die buiten de bestaande $spacer's vallen naar de dichtsbijzijnde waarde, i.e.

```suggestion
		padding-right: math.div($spacer-md, 2);
```

```suggestion
		padding-right: $spacer-sm;
```

---

Fixed